packages/forms-web-app/src/views/includes/head.njk

```diff
@@ -3,4 +3,14 @@
   <!--[if lte IE 8]><link href="/public/stylesheets/main-ie8.css" rel="stylesheet" type="text/css" /><![endif]-->
   <!--[if gt IE 8]><!--><link href="/public/stylesheets/main.css" media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
 
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script data-cy="Google Analytics" async src="https://www.googletagmanager.com/gtag/js?id=G-FFM8G1FWCZ"></script>
```

Comment:
Can we put the id in `config.js`?

Reply:
We could but from what I understood the vault is descoped and it will be one key for all the environments. So I'm not sure we need to make this configurable. Am I missing something ?

Reply:
Yes, it's been descoped as putting as an envvar, but I still think this should be stored in the `config.js` so it's a single source of truth. Something like:
```
module.exports = {
  server: {
    googleAnalytics: 'G-FFM8G1FWCZ'  
  }
}
```

At the very least, as this value is used twice in the template, I think it would be better putting to a variable - to my mind, getting that variable from `config.js` is as good a place as any to store it